x/docs/client/cli/query.go

```diff
@@ -117,3 +122,49 @@ func GetCmdSharedDocuments(cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+// ----------------------------------
+// --- DocumentReceipt
+// ----------------------------------
+
+func GetCmdReceivedReceipts(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "receipts [user-address]",
+		Short: "Get all the documents receipts sent to the given user address",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+
+			route := fmt.Sprintf("custom/%s/%s/%s", types.QuerierRoute, types.QueryReceipts, args[0])
+			res, _, err2 := cliCtx.QueryWithData(route, nil)
+			if err2 != nil {
+				fmt.Printf("Could not get any receipt associated with %s: \n %s", args[0], err2)
+			}
+
+			fmt.Printf(string(res))
+
+			return nil
+		},
+	}
+}
+
+func GetCmdReceivedReceiptByUuid(cdc *codec.Codec) *cobra.Command {
```

Comment:
I think `GetCmdReceivedReceiptByUuid` and `GetCmdReceivedReceipts` can be merged into a single command such as 

```golang
func GetCmdReceivedReceipts(cdc *codec.Codec) *cobra.Command {
	return &cobra.Command{
		Use:   "receipts [user-address] [[doc-uuid]]",
		Short: "Get all the documents receipts sent to the given user address",
		Args:  cobra.RangeArgs(1, 2),
		RunE: func(cmd *cobra.Command, args []string) error {
			cliCtx := context.NewCLIContext().WithCodec(cdc)

 			route := fmt.Sprintf("custom/%s/%s/%s", types.QuerierRoute, types.QueryReceipts, args[0])
                        // TODO: Handle the case where we have two arguments. I.e. add the last one to the end of the query

			res, _, err2 := cliCtx.QueryWithData(route, nil)
			if err2 != nil {
				fmt.Printf("Could not get any receipt associated with %s: \n %s", args[0], err2)
			}

 			fmt.Printf(string(res))

 			return nil
		},
	}
}
```

If the user